content/References/IPv6.md

```diff
@@ -0,0 +1,239 @@
+An example of an IPv6 address is *fe80:cd00:0000:0cde:1257:0000:211e:729c*.
+
+IPv6 addresses are split into two parts: a network (routing prefix) and a node component(interface ID).
+
+The network component is the first 64 bits of the address and is used for routing.
+This component can be split into a block of 48 bits for global network addresses and a block of 16 bits controlled by network administrators for subnets on an internal network.
+
+The node component is the second 64 bits of the IPv6 address and is used to identify the address of the interface.
+It is derived from the physical or MAC address, using the 64-bit extended unique identifier (EUI-64) format defined by the Institute of Electrical and Electronics Engineers (IEEE).
+This format designed to provide the ability to provide unique addresses to any electronic device equipped with sensors and connect them for data sharing.
+
+The IPv6 total address can be shortened. The addressing scheme allows the omission of any leading zero as well as any sequences consisting of only zeros.
+Using the example above, a shorted address looks like *fe80:cd00:0:cde:1257:0:211e:729c*.
+
+The specific layout of IPv6 addresses can vary depending on its format, with three basic parts: routing prefix, subnet ID, and the interface ID.
+The routing prefix and subnet ID represent two main levels in the address which is either global or site specific.
+Internet Registries and Internet service providers (ISPs) can decide to subdivide the routing prefix bits.
+The left most set of numbers or first 48 bits is the site prefix, and the next 16 bits is the subnet ID. The subnet ID lays out site topology.
+The 64 bit interface ID can be manually or automatically configured.
+
+IPv6 provides 1028 times more addresses than IPv4, which is running out of numbers due to the ever increasing number devices world wide wanting to connect.
+
+The rise of the Internet of Things (IoT), where connecting smart sensors in homes, motor vehicles, and businesses to the Internet allows data collection and sharing for analytics, device maintenance and monitoring, cost controls, increased safety and productivity, and overall improvement in consumer and customer experiences.
+
+## What are the Benefits of IPv6?
+Key benefits to IPv6 include eliminating the need for:
+
+* Network address translation (NAT)
+
+  IPv6 eliminates the need for NAT.
+  The structure of the 128 bit IPv6 address does not need the NAT method of remapping one IP address space into another by modifying network address information in the IP header of packets while in transit across a traffic routing device.
+
+* Private address collisions
+
+  Private address collisions occur when the same IP address is assigned to two different devices within the same private network, which can happen if IP addresses are not properly managed.
+  Using DHCP-assigned addresses is one way to manage IP address assignments to prevent assigning an IPv4 address to two different devices.
```

Comment:
This statement is true, but doesn't seem to be about IPv6.

Reply:
Reworked and moved line 35 up to a generalized statement about address demands.

Not sure what the issue is with lines 37-47. Reworded line 38.